.github/CODEOWNERS

```diff
@@ -106,7 +106,7 @@
 /sdk/apimanagement/Microsoft.Azure.Management.ApiManagement/       @miaojiang
 
 # PRLabel: %App Configuration
-/sdk/appconfiguration/                                             @nisha-bhatia @ShivangiReja @AlexanderSher @jsquire
+/sdk/appconfiguration/                                             @m-redding @ShivangiReja @AlexanderSher @jsquire
```

Comment:
```suggestion
/sdk/appconfiguration/                                             @ShivangiReja @jsquire
```

Reply:
Let's leave just myself and Shivangi for now, please.

Reply:
Done!